packages/spl/src/reward-manager/constants.ts

```diff
@@ -8,3 +8,24 @@ export enum RewardManagerInstruction {
   SubmitAttestation = 6,
   EvaluateAttestations = 7
 }
+
+/**
+ * Possible custom error messages from the Reward Manager program
+ * @see {@link https://github.com/AudiusProject/audius-protocol/blob/2a37bcff1bb1a82efdf187d1723b3457dc0dcb9b/solana-programs/reward-manager/program/src/error.rs solana-programs/reward-manager/program/src/errors.rs}
+ */
+export enum RewardManagerErrorCode {
+  IncorrectOwner = 0,
```

Comment:
just curious why the first one has `=0` and the rest don't

Reply:
they all inc from the starting number